src/coreclr/jit/lsrariscv64.cpp

```diff
@@ -998,6 +992,11 @@ int LinearScan::BuildCall(GenTreeCall* call)
     buildInternalRegisterUses();
 
     // Now generate defs and kills.
+    if (call->IsAsync() && compiler->compIsAsync() && !call->IsFastTailCall())
+    {
+        MarkAsyncContinuationBusyForCall(call);
+    }
+
```

Comment:
Should another `MarkAsyncContinuationBusyForCall` below be deleted?

Reply:
Yes, good catch